service/settings.py

```diff
@@ -264,7 +264,7 @@ def getInstance(cls):
     def __init__(self):
         serviceHTMLExportDefaultSettings = {
             "enabled": False,
-            "path"   : config.pyfaPath + os.sep + 'pyfaFits.html',
+            "path"   : os.path.join(config.savePath, 'pyfaFits.html'),
```

Comment:
It's actually not a Windows issue but a `os.path` issue.

If you pass a string (type of `str`) into `os.path`, and the string has unicode chars in it, it doesn't know how to handle it.

_Theoretically_, and this has only been proven in some isolated tests, all we have to do is:
`os.path.join(unicode(config.savePath), 'pyfaFits.html'),`